project/Build.scala

```diff
@@ -1176,6 +1193,17 @@ object Build extends sbt.Build {
     compiler % "plugin", library, jUnitRuntime % "test", jasmineTestFramework % "test"
   )
 
+  lazy val testSuiteJVM = Project(
+    id = "testSuiteJVM",
+    base = file("test-suite/shared"),
```

Comment:
Oh, because you put `testSuiteJVM` directly in `shared` ... That does not seem very clean. I'd rather `testSuiteJVM` was rooted in `test-suite/jvm`, and that you do add the shared files all the time.